make.sh

```diff
@@ -949,6 +956,7 @@ ci_setup_deps() {
     DEBIAN_FRONTEND=noninteractive pkg_setup_locale
     DEBIAN_FRONTEND=noninteractive pkg_install_llvm
     DEBIAN_FRONTEND=noninteractive pkg_install_rust
+    DEBIAN_FRONTEND=noninteractive pkg_install_solc_linux
```

Comment:
CI pipeline for installing packages should be for setting up CI runner with necessary deps to run the build workflow. Target architectures differ for the multi-arch build pipeline for our supported architectures, so target specific deps should not be here. Target specific dependencies should be shifted inside build-depends pipeline, which will install the correct target dependencies based on the specified target architecture.